jwst/scripts/adjust_wcs.py

```diff
@@ -116,96 +115,71 @@ def angle(arg):
-        type=str,
-        help="Input data model file(s)."
+        "arg0", nargs="*", metavar="input", type=str, help="Input data model file(s)."
     )
 
     group = parser.add_mutually_exclusive_group(required=True)
-    group.add_argument(
-        "-u",
-        "--update",
-        action="store_true",
-        help="Update input data models."
-    )
-    group.add_argument(
-        "--suffix",
-        type=str,
-        help="Suffix for output files."
-    )
-    group.add_argument(
-        "-f",
-        "--file",
-        type=str,
-        help="Output file name."
-    )
+    group.add_argument("-u", "--update", action="store_true", help="Update input data models.")
+    group.add_argument("--suffix", type=str, help="Suffix for output files.")
+    group.add_argument("-f", "--file", type=str, help="Output file name.")
 
     parser.add_argument(
         "--overwrite",
         action="store_true",
-        help="Overwrite existing output files (ignored with -u/--update)."
+        help="Overwrite existing output files (ignored with -u/--update).",
     )
 
-    group_wcs = parser.add_argument_group(
-        title='WCS adjustment parameters'
-    )
-    group_wcs.add_argument(
-        "-r",
-        "--ra_delta",
-        type=angle,
-        default=0.0,
-        help="Delta RA (longitude), degrees."
-    )
+    group_wcs = parser.add_argument_group(title="WCS adjustment parameters")
     group_wcs.add_argument(
-        "-d",
-        "--dec_delta",
-        type=angle,
-        default=0.0,
-        help="Delta DEC (latitude), degrees."
+        "-r", "--ra_delta", type=angle, default=0.0, help="Delta RA (longitude), degrees."
     )
     group_wcs.add_argument(
-        "-o",
-        "--roll_delta",
-        type=angle,
-        default=0.0,
-        help="Delta roll angle, degrees."
+        "-d", "--dec_delta", type=angle, default=0.0, help="Delta DEC (latitude), degrees."
     )
     group_wcs.add_argument(
-        "-s",
-        "--scale_factor",
-        type=float,
-        default=1.0,
-        help="Scale factor."
+        "-o", "--roll_delta", type=angle, default=0.0, help="Delta roll angle, degrees."
     )
+    group_wcs.add_argument("-s", "--scale_factor", type=float, default=1.0, help="Scale factor.")
 
-    parser.add_argument(
-        "-v",
-        "--version",
-        action="version",
-        version="v{0}".format(jwst.__version__)
-    )
+    parser.add_argument("-v", "--version", action="version", version=f"v{jwst.__version__}")
 
     # pre-process argv for units and negative floats:
     argv = sys.argv[1:]
-    argv_new = [os.path.basename(sys.argv[0])]
+    argv_new = [Path(sys.argv[0]).name]
     while argv:
         argv_new.append(argv.pop(0))
 
         # check whether next argument is a float:
-        if (argv_new[-1] in _ANGLE_PARS and len(argv) >= 1 and
-                _is_float(argv[0])):
+        if argv_new[-1] in _ANGLE_PARS and len(argv) >= 1 and _is_float(argv[0]):
```

Comment:
It might enhance readability to add parentheses around each of the three conditionals